man/column-configuration.md

```diff
@@ -37,6 +37,28 @@ const columns = [
   ...
 ```
 
+#### Column header hints
+
+  You can specify a `hint` string for the label on the column header.
+  If the `hint` field is not provided, the `label` will be used instead, if `label` is not provided, `key` will be used
+  If `hint` is specified as empty string then hint will not be displayed
```

Comment:
If you have to explicitly set `hint` to an empty string in order to disable it, then this is a breaking change – no?

Reply:
No, if `hint` is missing then old logic is applied